libs/saksbehandling-common/src/main/kotlin/person/Person.kt

```diff
@@ -193,14 +193,14 @@ interface Verge {
     fun navn(): String?
 }
 
-data class Vergemaal(val mottaker: BrevMottaker) : Verge {
+data class Vergemaal(val navn: String?, val foedselsnummer: Folkeregisteridentifikator) : Verge {
     override fun navn(): String? {
-        return mottaker.navn
+        return navn
     }
 }
 
 data class ForelderVerge(val foedselsnummer: Folkeregisteridentifikator, val navn: String) : Verge {
-    override fun navn(): String {
+    override fun navn(): String { // TODO ble denne brukt?
```

Comment:
fant du ut av dette? eventuelt slette den og se om den har noe bruk?

Reply:
Takk :) Det var vel litt fram og tilbake her. Jeg la til et nytt kall til adresseService i mapVerge() for å få med navnet (uten å ta vare på hele adressen), slik at eksisterende kode kan bruke navn() på samme måte som de gjør per nå.